calico-enterprise/getting-started/install-on-clusters/kubernetes/helm.mdx

```diff
@@ -64,13 +69,7 @@ If you are installing on a cluster installed by EKS, GKE, AKS or Mirantis Kubern
   EOF
   ```
 
-1. Add any other customizations you require to `values.yaml`. To see values that can be customized in the chart, see the [helm docs](https://helm.sh/docs/) or run the following command:
-
-  <CodeBlock language='bash'>
```

Comment:
Why are we removing the section indicating how to examine your `values.yaml`?

Reply:
1. The link to helm customization doc page is missing from this paragraph and we have general helm docs page instead (not sure how general docs can help users figure out how they can customize their calico installation values.yaml)

2. The command in the docs ``helm show values ./tigera-operator-{{chart_version_name}}.tgz`}` just shows the current values set in valuels.yaml. I am again not sure how that helps users figure out what are the options they can define in values.yaml and how they can initialize them.

So I just replaced this whole section with:
```
For more information about configurable options via values.yaml please see [Helm installation reference](../../reference/installation/helm_customization).
```